garage/theano/envs/base.py

```diff
@@ -1,51 +1,45 @@
 """Wrapper class that converts gym.Env into TheanoEnv."""
-from cached_property import cached_property
```

Comment:
uhhh the properties were cached for a reason. please keep them cached.

Reply:
How do I cache variables like `self.observation_space`, `self.action_space`, rather than functions (as it's used here)

Reply:
You make them properties